### DISCUSSION TITLE
support for Window 7 and Windows 8

### DISCUSSION THREAD

**Discussion by UnfortunateLiama** [Chatterino/chatterino2#4852] [Q&A] [answered]
Can you please reconsider dropping support for older windows ?
https://www.qt.io/blog/two-qt-security-advisorys-gdi-font-engine-webp-image-format
seems to provide security patches to Qt 5.15

Thank you

**Comment by Felanbird** [comment]
The [nightly build](https://github.com/Chatterino/chatterino2/releases/tag/nightly-build) that uses qt5.15 still exists, and if you're on Windows 7 you shouldn't even be connected to the internet, so I doubt you care about this security issue.

**Comment by Mm2PL** [comment]
Are there builds of Qt with these patches even available? Building Qt from source is an endeavor, we probably won't be taking to support these EOL operating systems.

**Reply by Felanbird** [reply]
No we'd have to build it ourselves which is why we didn't

**Comment by pajlada** [comment, marked as answer]
For users who are stuck with Windows 7 or Windows 8.1, I'd suggest either staying on 2.4.5 for now, or using the latest Nightly version. We'll keep building Qt 5.15 builds for now, but you should know they won't have security fixes applied.

The effort required to be backwards-compatible with this support is too high for me to look into, but I'd be open to a PR looking to fix it.

You should really consider upgrading to Windows 10 if possible, not for Chatterino alone, but because your system hasn't received any security patches in almost 4 years now

**Reply by Felanbird** [reply]
As mentioned the [nightly build](https://github.com/Chatterino/chatterino2/releases/tag/nightly-build) missing the security fix is the one named `chatterino-windows-old-x86-64-Qt-5.15.2.zip` - this will be the only version supporting Windows 7/8, and will only exist until we stop supporting `qt5.15.2`.

**Comment by fourtf** [comment]
While we can tackle the referenced compatibility issue with some effort, it hints at future security and compatibility concerns. This is not the last problem that we will encounter while trying to work with outdated software. Eventually, we must discontinue support for older operating systems, and if the vendor no longer supports them, then that's a good reason to not go out of our way to support it. In my own opinion, I think the writing is on the wall.

**Comment by Felanbird** [comment]
As mentioned above the
